<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="M5-L-5194G16" name="com.gurukula.tests.RegistrationTest" tests="22" failures="4" timestamp="10 Jul 2018 01:45:09 GMT" time="46.976" errors="0">
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.601" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.771" classname="com.gurukula.tests.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [Your e-mail cannot be longer than 50 characters.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Your e-mail cannot be longer than 50 characters.] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.gurukula.tests.RegistrationTest.M2_TS03_Registeration_ErrorMessageVerification(RegistrationTest.java:102)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- M2_TS03_Registeration_ErrorMessageVerification -->
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.125" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS02_VerifyPasswordStrength" time="2.938" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.219" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.281" classname="com.gurukula.tests.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [Password should begin with a alphabet and should contain a number and a special character.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Password should begin with a alphabet and should contain a number and a special character.] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.gurukula.tests.RegistrationTest.M2_TS03_Registeration_ErrorMessageVerification(RegistrationTest.java:137)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- M2_TS03_Registeration_ErrorMessageVerification -->
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.308" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS02_VerifyPasswordStrength" time="4.268" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.120" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.331" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.024" classname="com.gurukula.tests.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [Password should begin with a alphabet and should contain a number and a special character.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Password should begin with a alphabet and should contain a number and a special character.] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.gurukula.tests.RegistrationTest.M2_TS03_Registeration_ErrorMessageVerification(RegistrationTest.java:119)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- M2_TS03_Registeration_ErrorMessageVerification -->
  <testcase name="M2_TS04_VerifyRegistrationEnable" time="2.731" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.226" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS02_VerifyPasswordStrength" time="2.661" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.061" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS01_Verify_NewUser_Registration" time="8.313" classname="com.gurukula.tests.RegistrationTest">
    <failure type="java.lang.AssertionError" message="expected [Registration Success!] but found [Registration failed!]">
      <![CDATA[java.lang.AssertionError: expected [Registration Success!] but found [Registration failed!]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.gurukula.tests.RegistrationTest.M2_TS01_Verify_NewUser_Registration(RegistrationTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- M2_TS01_Verify_NewUser_Registration -->
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.828" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.199" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS02_VerifyPasswordStrength" time="2.545" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="2.082" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="1.207" classname="com.gurukula.tests.RegistrationTest"/>
  <testcase name="M2_TS03_Registeration_ErrorMessageVerification" time="3.137" classname="com.gurukula.tests.RegistrationTest"/>
</testsuite> <!-- com.gurukula.tests.RegistrationTest -->
